riscv/sim.cc

```diff
@@ -109,6 +109,21 @@ sim_t::sim_t(const cfg_t *cfg, const char* varch, bool halted, bool real_time_cl
     bus.add_device(clint_base, clint.get());
   }
 
+  // create plic
```

Comment:
I think this should be conditionalized similar to the CLINT (see above
https://github.com/riscv-software-src/riscv-isa-sim/blob/fc88987f4587556a8dbf184825263263b19c01e7/riscv/sim.cc#L107) so that users who pass in their own DeviceTree can choose not to instantiate the PLIC.

Reply:
okay, I will update.